Mathlib/CategoryTheory/Galois/Basic.lean

```diff
@@ -123,8 +127,77 @@ noncomputable instance : ReflectsColimitsOfShape (Discrete PEmpty.{1}) F :=
 noncomputable instance : PreservesFiniteLimits F :=
   preservesFiniteLimitsOfPreservesTerminalAndPullbacks F
 
+/-- Fibre functors reflect monomorphisms. -/
+instance foo : ReflectsMonomorphisms F := ReflectsMonomorphisms.mk <| by
```

Comment:
```suggestion
instance : ReflectsMonomorphisms F := ReflectsMonomorphisms.mk <| by
```
You can choose a name if you want.

Reply:
Good catch, I added this earlier to analyze @kbuzzard s Remark.